FacuReartes/react-laba-final-project-team-2
# SPT2-159 - Add debounce to price filter in home page

# Description


## Changes

- Updated the `FilterPrice` component to decrease data fetching while user still changing slider value.
- Added empty products to state to Products page

### Jira

- [SPT2-159](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-159?atlOrigin=eyJpIjoiZjgwNzc1MjkyY2FjNGU0ZThlMjliNWMxZTE3YTIxYTMiLCJwIjoiaiJ9)
- [SPT2-177](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-177?atlOrigin=eyJpIjoiOThkZGYxM2M1MTg0NDk0ZmE3MTlmMTIzY2RmYTE4OWUiLCJwIjoiaiJ9)

### How to Test

1. Navigate to ['http://localhost:3000/'](http://localhost:3000/).
2. Use filter to reach a no matches results to see empty results screen.
3. Open Navigation tab in devtools
4. Change prices filter
5. Check navigation tab fetch after a second you stopped change slider

### Screenshots

![image](https://github.com/user-attachments/assets/ca0f5373-db05-41bc-9728-8d7da029238c)

### Checklist



[SPT2-159]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-159?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SPT2-177]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
